test_chameleon.py

```diff
@@ -13,14 +13,14 @@ def test_basic_json_service(self):
         responds to queries as expected.
         Port defaults to 8080.
         """
-        settings_list = [{"url": "/path1", 
-                          "status_code": 200, 
-                          "content_type": "application/json", 
+        settings_list = [{"url": "/path1",
+                          "status_code": 200,
+                          "content_type": "application/json",
                           "content": '{"foo": "bar"}'}]
         chameleon.start(settings_list)
         response = requests.get('http://localhost:8080/path1')
         self.assertEqual(response.status_code, 200)
-        self.assertEqual(response.content, '{"foo": "bar"}')
+        self.assertEqual(response.content, b'{"foo": "bar"}')
```

Comment:
what is this `b'string'` thing?  Am I still going to be able to use this with 2.7?